src/qz/printer/status/CupsStatusHandler.java

```diff
@@ -73,6 +87,7 @@ private void parseXML(XMLEventReader eventReader) throws XMLStreamException {
                         } else {
                             String printerName = StringUtils.substringBeforeLast(description, "\"");
                             printerName = StringUtils.substringAfter(printerName, "\"");
+                            printerName = StringEscapeUtils.unescapeXml(printerName);
```

Comment:
These lines can be combined:
```java
printerName = StringEscapeUtils.unescapeXml(StringUtils.substringAfter(printerName, "\""));
```

Reply:
Wouldn't it have to be `StringUtils.substringBeforeLast(StringEscapeUtils.unescapeXml(StringUtils.substringAfter(printerName, "\"")), "\"");`?

Reply:
Yeah, that's ufly.  Perhaps `substringBetween` instead?

Reply:
I think the matches from left to right, we need to do an outer match.

Reply:
> I think the matches from left to right, we need to do an outer match.

You're right, and MacOS allows double quotes.  We should leave it as-is.